#rec1_distance                  : beacon distance  (m) (int)     
#rec1_direction                 : Signal direction (deg) (int) 
#rec2_distance                  : Signal distance  (m) (int)     
#rec2_direction                 : Signal direction (deg) (int) 
#rec3_distance                  : Signal distance  (m) (int)     
#rec3_direction                 : Signal direction (deg) (int) 
#rec4_distance                  : Signal distance  (m) (int)     
#rec4_direction                 : Signal direction (deg) (int) 

Header header
int32 rec1_distance
int32 rec1_direction
int32 rec2_distance
int32 rec2_direction
int32 rec3_distance
int32 rec3_direction
int32 rec4_distance
int32 rec4_direction